src/Control/Bind.purs

```diff
@@ -73,11 +73,11 @@ foreign import arrayBind :: forall a b. Array a -> (a -> Array b) -> Array b
 -- |
 -- | An example is the `Unit` type, since there is only one
 -- | possible value which can be returned.
-class Discard a where
-  discard :: forall f b. Bind f => f a -> (a -> f b) -> f b
+class BindDiscard a where
+  bind_ :: forall f b. Bind f => f a -> (a -> f b) -> f b
```

Comment:
Well, I started it! :wink: I agree that `bind_` doesn't quite seem right after all, since it's not giving back a `Unit` result like the other cases; that's down to a misunderstanding I had of how it was working.

Re: rebindable syntax, would one even need to redefine `bind_`/`discard`/whatever if overriding `bind`? Or only if they were doing something "fancy"?